h2gis-functions/src/test/java/org/h2gis/functions/io/geojson/GeojsonImportExportTest.java

```diff
@@ -866,6 +866,46 @@ public void exportQueryImportFile() throws SQLException, IOException {
         }
     }
 
+    @Test
+    public void exportQueryImportFileOption() throws SQLException, IOException {
+        Statement stat = connection.createStatement();
+        File fileOut = new File("target/lineal_export.geojson");
+        stat.execute("DROP TABLE IF EXISTS LINEAL");
+        stat.execute("create table lineal(idarea int primary key, the_geom GEOMETRY(LINESTRING Z))");
+        stat.execute("insert into lineal values(1, 'LINESTRING(-10 109 5, 12 2 6)'),(2, 'LINESTRING(-15 109 5, 120 2 6)')");
+        // Create a geojson file using a query
+        stat.execute("CALL GeoJSONWrite('target/lineal_export.geojson', '(SELECT THE_GEOM FROM LINEAL LIMIT 1 )', true)");
+        // Read this shape file to check values
+        assertTrue(fileOut.exists());
+        stat.execute("DROP TABLE IF EXISTS IMPORT_LINEAL;");
+        stat.execute("CALL GeoJSONRead('target/lineal_export.geojson', 'IMPORT_LINEAL', true)");
+
+        try (ResultSet res = stat.executeQuery("SELECT COUNT(*) FROM IMPORT_LINEAL;")) {
+            res.next();
+            assertEquals(1, res.getInt(1));
+        }
+    }
+
+    @Test
+    public void exportQueryImportFileGZOption() throws SQLException, IOException {
+        Statement stat = connection.createStatement();
+        File fileOut = new File("target/lineal_export.gz");
+        stat.execute("DROP TABLE IF EXISTS LINEAL");
+        stat.execute("create table lineal(idarea int primary key, the_geom GEOMETRY(LINESTRING Z))");
+        stat.execute("insert into lineal values(1, 'LINESTRING(-10 109 5, 12 2 6)'),(2, 'LINESTRING(-15 109 5, 120 2 6)')");
+        // Create a geojson file using a query
+        stat.execute("CALL GeoJSONWrite('target/lineal_export.gz', '(SELECT THE_GEOM FROM LINEAL LIMIT 1 )', true)");
+        // Read this shape file to check values
+        assertTrue(fileOut.exists());
+        stat.execute("DROP TABLE IF EXISTS IMPORT_LINEAL;");
+        stat.execute("CALL GeoJSONRead('target/lineal_export.gz')");
+
+        try (ResultSet res = stat.executeQuery("SELECT COUNT(*) FROM LINEAL_EXPORT;")) {
```

Comment:
Be careful, if `stat.executeQuery` fails with an exception, failing won't be detected.